tests/unit/workflow/test_workflow_subgraphs.py

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import pytest
+
+import nvtabular.ops as ops
+from merlin.dag import ColumnSelector
+from merlin.dag.ops.subgraph import Subgraph
+from nvtabular import Workflow
+
+
+@pytest.mark.parametrize("engine", ["parquet"])
```

Comment:
These tests check that the normal graph API that works on other operators also works as expected with the `Subgraph` operator. They're easier to test here than in Core because we have operators that effect actual transforms here, but I'm starting to think that the Rename and Lambda operators might be appropriate to move to Core as part of the generic DAG operators that can be used in all Merlin DAGs.